gc-check-ins.md

```diff
@@ -0,0 +1,20 @@
+## Regular check-ins with SIG leads
+
+As defined in the [charter of the GC](./governance-charter.md#regular-check-ins-with-sig-leads) the GC will check in with maintainers of all long-term and short-term SIGs on a regular basis. During this check in,
+
+* any project files or roadmap entries should be reviewed to verify that they are still accurate.
+* if a SIG lead has any private concerns they wish to raise with the Governance Committee, this is an opportunity to do so.
+* the health of the SIG will be reviewed, and what measures can be applied to help the SIG.
+
+The check in process is as follows:
+
+* For each SIG a GC member is appointed as a GC liaison. The name of that person will be listed publicly.
+* If a GC member is the maintainer themselves, they are by default the GC liaison for this SIG.
```

Comment:
Newer SIGs follow the requirement to have at least two TC/GC sponsors, such as the Security SIG. Should the GC sponsor be the liaison in that case?

Reply:
I added some wording on this topic